awx/ui/src/screens/Inventory/shared/InventorySourceForm.js

```diff
@@ -207,7 +207,7 @@ const InventorySourceFormFields = ({
                     }
                   />
                 ),
-                tower: (
+                controller: (
                   <TowerSubForm
```

Comment:
Should we change the name of the form?  I feel like at some point we should remove references to tower.  Maybe a backlog issue or something